orttraining/orttraining/test/python/orttraining_run_bert_pretrain.py

```diff
@@ -673,19 +738,7 @@ def test_pretrain_zero(self):
         test.world_rank = local_rank
         test.world_size = world_size
 
-        if len(sys.argv) >= 2 and sys.argv[1] == 'ORTBertPretrainTest.test_pretrain_throughput':
-            logger.info("running ORTBertPretrainTest.test_pretrain_throughput()...")
-            test.test_pretrain_throughput()
-            logger.info("ORTBertPretrainTest.test_pretrain_throughput() passed")
-        elif len(sys.argv) >= 2 and sys.argv[1] == 'ORTBertPretrainTest.test_pretrain_convergence':
-            logger.info("running ORTBertPretrainTest.test_pretrain_convergence()...")
-            test.max_steps = 200
-            test.force_num_hidden_layers = 8
-            final_loss = test.test_pretrain_convergence()
-            logger.info("ORTBertPretrainTest.test_pretrain_convergence() final loss = %f", final_loss)
-            test.assertLess(final_loss, 8.5)
-            logger.info("ORTBertPretrainTest.test_pretrain_convergence() passed")
-        elif len(sys.argv) >= 2 and sys.argv[1] == 'ORTBertPretrainTest.test_pretrain_zero':
+        if len(sys.argv) >= 2 and sys.argv[1] == 'ORTBertPretrainTest.test_pretrain_zero':
```

Comment:
Consider using `argparse` as opposed to manipulate `argv` directly

Reply:
unit test is against passing arguments so we need sys.argv to work around it. As you can see I still do not have a good solutions for it - I am not sure if a solution is needed for this artificial restriction of not allowing arguments to an unit test.